### PR TITLE
Add argh recipe

### PR DESCRIPTION
**Pull request by goanpeca**

### CODE REVIEW

File: recipes/argh/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    - python
+    - argparse >= 1.1  # [py26]
+
+test:
+  imports:
+    - argh
+
+about:
+  home: https://github.com/neithere/argh
+  license: GNU Lesser General Public License (LGPL), Version 3
```
Comment (jakirkham) [inline recipes/argh/meta.yaml]: Please shorten to `LGPL 3`.

File: recipes/argh/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  summary: The Natural CLI.
```
Comment (jakirkham) [inline recipes/argh/meta.yaml]: Also have asked upstream to include the license files in packages via PR ( https://github.com/neithere/argh/pull/112 ). Please keep an eye on this via an issue in the feedstock. Once it gets corrected in a version we should add the license via `license_file`.

**Comment by jakirkham** [comment]
Basically fine. One minor comment. Otherwise LGTM.

File: recipes/argh/meta.yaml (hunk 2 above)
Comment (goanpeca) [inline recipes/argh/meta.yaml]: Yes, I noticed the PR, should we use **COPYING**, or **COPYING.LESSER** file ?

Comment (jakirkham) [inline recipes/argh/meta.yaml]: Good question. My inclination would be to use `COPYING.LESSER` as this is the LGPL license file.

Comment (jakirkham) [inline recipes/argh/meta.yaml]: In any event, we don't need to do anything with this before it is a feedstock.

**Comment by jakirkham** [comment]
Thanks @goanpeca.

File: recipes/argh/meta.yaml (hunk 2 above)
Comment (jakirkham) [inline recipes/argh/meta.yaml]: Raised issue ( https://github.com/conda/conda-build/issues/1247 ) to get more info on ways to package multiple license files.

**Pull request merged**